service/frontend/operator_handler.go

```diff
@@ -55,26 +56,26 @@ type (
 	OperatorHandlerImpl struct {
 		status int32
 
-		healthStatus  int32
-		logger        log.Logger
-		esConfig      *esclient.Config
-		esClient      esclient.Client
-		sdkClient     sdkclient.Client
-		metricsClient metrics.Client
-		saProvider    searchattribute.Provider
-		saManager     searchattribute.Manager
-		healthServer  *health.Server
+		healthStatus     int32
+		logger           log.Logger
+		esConfig         *esclient.Config
+		esClient         esclient.Client
+		sdkClientFactory sdk.ClientFactory
+		metricsClient    metrics.Client
+		saProvider       searchattribute.Provider
+		saManager        searchattribute.Manager
+		healthServer     *health.Server
 	}
 
 	NewOperatorHandlerImplArgs struct {
```

Comment:
weird that this looks like fx init params but does not have a fx.In embeded.

Reply:
None of the handlersArgs are embedding fx.In.

Reply:
It looks like this pattern might be an oversight.

The constructor functions take individual args instead of the *Args struct and that is was fx.In is not embedded.

Reply:
sgtm